spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/README.adoc

```diff
@@ -1,62 +0,0 @@
-= Google Cloud Core Spring Boot Starter
```

Comment:
This file can't also just disappear.
If `spring-cloud-gcp-core` is becoming a starter on its own, then this should probably go there.